### PR TITLE
[48] Maven-specific post/pre-publish code still in common plugin

### PR DESCRIPTION
**Pull request by keithchong**
PR for #48

### CODE REVIEW

**Comment by keithchong** [comment]
@eharris369 , please review.

File: dev/com.ibm.ws.st.liberty.buildplugin.integration/src/com/ibm/ws/st/liberty/buildplugin/integration/servertype/internal/AbstractLibertyBuildPluginJEEPublisher.java [hunk 1]
```diff
@@ -107,7 +101,7 @@ public void prePublishApplication(int kind, PublishUnit app, MultiStatus status,
 
         if (wsServer != null) {
             String type = wsServer.getServerType();
-            if (Constants.SERVER_TYPE_LIBERTY_MAVEN.equals(type)) {
+            if (Constants.SERVER_TYPE_LIBERTY_MAVEN.equals(type) || Constants.SERVER_TYPE_LIBERTY_GRADLE.equals(type)) {
```
Comment (eharris369) [inline dev/com.ibm.ws.st.liberty.buildplugin.integration/src/com/ibm/ws/st/liberty/buildplugin/integration/servertype/internal/AbstractLibertyBuildPluginJEEPublisher.java]: For postPublishApplication it is overriding in the Maven specific code but for prePublishApplication it is not?

Comment (eharris369) [inline dev/com.ibm.ws.st.liberty.buildplugin.integration/src/com/ibm/ws/st/liberty/buildplugin/integration/servertype/internal/AbstractLibertyBuildPluginJEEPublisher.java]: Have the new constants for Gradle been delivered yet?

**Comment by keithchong** [comment]
Hi @eharris369, the new Constant was meant to be delivered with the new plugin.  I wanted this refactoring-related changed delivered first since it is supposed to be independent.  The prePublish method was supposed to have the similar change as the postPublish method.   I've updated the changes now.  Please have a look.

**Comment by keithchong** [comment]
fixes #48

**Pull request merged**